hoot-core/src/main/cpp/hoot/core/schema/OsmSchema.cpp

```diff
@@ -509,7 +509,7 @@ class OsmSchemaData
 
   const SchemaVertex& getFirstCommonAncestor(const QString& kvp1, const QString& kvp2)
   {
-    if (!_name2Vertex.contains(kvp1) || !_name2Vertex.contains(kvp1))
+    if (!_name2Vertex.contains(kvp1) || !_name2Vertex.contains(kvp2))
```

Comment:
Here we have a minor slip of the keyboard, check for `_name2Vertex` in both key/value pairs instead of just the first one twice.